samples/sampler/stories/qa/MenuItem.module.less

```diff
@@ -0,0 +1,10 @@
+.menuItem {
```

Comment:
Panels.module.less
is more proper. because the less is only used on Panles.js

Reply:
I removed the file as it wasn't needed.